openedx/core/djangoapps/external_user_ids/docs/decisions/0002-programs-cache.rst

```diff
@@ -0,0 +1,83 @@
+Programs Cache in LMS
+_____________________
+
+Status
+======
+Pending
+
+Context
+=======
+The LMS in edx-platform has a limited understanding of the structure and content of programs, as they are stored authoritatively in the Discovery service's SQL database. However, certain features of the LMS, such as displaying a learner’s programs on their dashboard, require access to the programs’ data. To fulfill this need, the LMS has the JSON responses from Discovery’s /api/v1/programs endpoint stored in memcached. The LMS uses the cache to understand what program or programs a course (run) is in. The contents of the cache are exposed through the openedx.core.djangoapps.catalog app. This system is what we are referring to when we say the programs cache” or simply “the cache”.
```

Comment:
Can you mark openedx.core.djangoapps.catalog and cache_programs with double-back-ticks (\`\`like_this\`\`) so they appear as code?